kubectl-cloudflow/verify/streamletref_test.go

```diff
@@ -0,0 +1,83 @@
+package verify
+
+import (
+	"gotest.tools/assert"
+	"testing"
+)
+
+func Test_verifyStreamletRefWithNoProblems(t *testing.T) {
+	var ref = getTestStreamletRef()
+	var descriptors = []StreamletDescriptor{}
+	descriptors =  append(descriptors, StreamletDescriptor{
+		ClassName: "sensors.MovingAverageSparklet",
+		Runtime: "spark",
+	})
+	assert.Equal(t, len(ref.verify(descriptors).problems),0 , "Number of problems should be zero")
+}
+
+func Test_verifyStreamletRefWithAmbiguityProblem(t *testing.T) {
+	var ref = getTestStreamletRef()
+	var descriptors = []StreamletDescriptor{}
+	descriptors =  append(descriptors, StreamletDescriptor{
+		ClassName: "test1.sensors.MovingAverageSparklet",
+		Runtime: "spark",
+	})
+	descriptors =  append(descriptors, StreamletDescriptor{
+		ClassName: "test.sensors.MovingAverageSparklet",
+		Runtime: "spark",
+	})
+
+	assert.Equal(t, GetProblem(ref.verify(descriptors).problems, AmbiguousStreamletRef{
+		AmbiguousStreamletRef{},
+		ref.name,
+		ref.className,}), true)
+}
```

Comment:
How about https://github.com/lightbend/cloudflow/issues/164?

Reply:
Should be fixed, it can create some really confusing problems.

Reply:
Ok will do that

Reply:
yes please fix in this PR